### ISSUE TITLE
Dark theme link colours might not be readable/accessible

### ISSUE THREAD

**Issue by juanruitina** [canonical/vanilla-framework#4844]
The shade of blue (`#6699cc`) and purple (`#a679d2`) used for links in the dark theme might make text not readable enough. The colour combinations with the graphite background (`#262626`) definitely comply with the WCAG algorithm (AAA actually), but:

1. We received feedback [from a colleague](https://discourse.canonical.com/t/introducing-the-new-canonical-com/2204/3) that contrast is too low on canonical.com; I had the same intuition. Given contrast is heavily perceptual, users' perceptions should be taken into consideration.
2. The WCAG algorithm is known to throw false positives in many instances and for blues in particular ([one source](https://blogs.kent.ac.uk/webdev/2020/08/18/colours-contrast-and-accessibility/)), because blue has low luminance.
3. The combination fails the APCA algorithm ([blue on graphite](https://contrast.tools?text=6699cc&background=262626), [purple on graphite](https://contrast.tools?text=a679d2&background=262626)), particularly given the very thin font weight we are using. Even if it's not the official algorithm, I would take this as another sign something is wrong with the colour combination.

For context, here's [Github's own discussion around the topic](https://github.com/orgs/community/discussions/51233). They agree it's a tough one. I agree that "Visual task of differentiation is not all the same as the visual task of fluent readability" (a comment by the way made by the proponent of the APCA algorithm). It is a tradeoff, and I am on the side of ensuring readability, as differentiation can be achieved in other ways (e.g. positioning, underlining).

I acknowledge this is difficult to address, as we might need to choose between compliance and actual accessibility; this is something we have discussed a few times already. But wanted to document this nonetheless.

Some more reads: 
- https://github.com/w3c/low-vision-a11y-tf/issues/84
- [NASA research on designing with blue](https://colorusage.arc.nasa.gov/blue_2.php)

When did the issue start? Vanilla 4.0.0
To reproduce: visit [canonical.com](https://canonical.com), such links are used across the page and in the Careers navigation item.

![canonical com_(Desktop)](https://github.com/vanilla/vanilla/assets/1409729/42a31cee-e7d5-4ebe-b261-1a0b80df474b)
![canonical com_(Desktop) (1)](https://github.com/vanilla/vanilla/assets/1409729/79ec1727-976b-4dd1-ae65-fb3f1bac0b95)
